packages/contentstack-branches/src/branch/merge-handler.ts

```diff
@@ -66,6 +66,12 @@ export default class MergeHandler {
       await this.executeMerge(mergePayload);
     } else if (this.executeOption === 'export') {
       await this.exportSummary(mergePayload);
+    } else if (this.executeOption === 'merge_n_scripts') {
+      this.enableEntryExp = true;
+      await this.executeMerge(mergePayload);
+    } else if (this.executeOption === 'summary_n_scripts') {
+      this.enableEntryExp = true;
+      await this.exportSummary(mergePayload);
```

Comment:
Using a switch case could have streamlined the code instead of multiple if-else conditions. Additionally, the final else block and the initial if block share identical code.